cypress/helpers/cypressTasks/pdf/parsePdf.ts

```diff
@@ -1,6 +1,10 @@
-import * as pdfjsLib from 'pdfjs-dist/legacy/build/pdf.mjs';
 import { readFile } from 'fs/promises';
-pdfjsLib.GlobalWorkerOptions.workerSrc = './pdf.worker.mjs';
+
+async function getPdfJs(): Promise<typeof pdfJs> {
+  const pdfJs = await import('pdfjs-dist/legacy/build/pdf.mjs');
+  pdfJs.GlobalWorkerOptions.workerSrc = './pdf.worker.mjs';
+  return pdfJs;
+}
```

Comment:
Because of (it seems) [this commit](https://github.com/mozilla/pdf.js/pull/19689) in [the most recent release of pdfjs](https://github.com/mozilla/pdf.js/releases/tag/v5.1.91), we need to make sure to lazy load the pdfjs library, otherwise:

1. Cypress will load this file
2. It will then attempt to load pdfjs
3. pdfjs will look for browser-specific stuff ... but that stuff isn't here yet because the browser isn't running
4. Fail